Mercury-Store/Mercury-Store/Profile/Views/RegisterView/RegisterViewController.swift

```diff
@@ -45,8 +58,8 @@ class RegisterViewController: UIViewController,ProfileCoordinated{
     }
     */
     @IBAction func signUp(_ sender: Any) {
-        
-    coordinator?.moveTo(flow: .profile(.registerScreen), userData: nil)
+    print("Tapped Login Button")
+    coordinator?.moveTo(flow: .profile(.intialScreen), userData: nil)
```

Comment:
States should be private since it's related to the coordinator. My idea here it to create API for showing profile as initial screen `func showUserProfile()`.. This will make coordinator know only why and how to handle the coordination and no outer view can know this info.

Reply:
we will change theses and make review again